### PR TITLE
Logout

### PR DESCRIPTION
**Pull request by Moniarchy**
@jrob8577 "lol"

### CODE REVIEW

File: readhoney/routes/users.js [hunk 1]
```diff
@@ -38,9 +39,12 @@ router.post ( '/signup', (request, response, next) => {
 
 
 router.get('/logout', (request, response) => {
-  request.logout()
-  response.redirect('/')
+    request.logout();
+    delete request.session;
```
Comment (jrobcodes) [inline readhoney/routes/users.js]: This is interesting. I don't know what this does, teach me? 😄 (I know that it deletes the key session from the request object, but that would only do it within the context of this request.  I suspect that the `logout` function does something like this in order to ensure the session is destroyed, but would love to learn about why/how this works.)

File: readhoney/views/index.pug [hunk 2]
```diff
@@ -5,9 +5,10 @@ block content
   h1= title
   h1 Welcome to #{title}
   h3 You ARE HERE
+  
+  a(href='/logout') Logout
```
Comment (jrobcodes) [inline readhoney/views/index.pug]: Should this be `/users/logout`?

File: readhoney/routes/users.js (hunk 1 above)
Comment (Moniarchy) [reply readhoney/routes/users.js]: We'd be happy to try!

Comment (jrobcodes) [reply readhoney/routes/users.js]: K, I'll come check in after this meeting.

File: readhoney/views/index.pug (hunk 2 above)
Comment (Moniarchy) [reply readhoney/views/index.pug]: Right now, the code is having the effect we want it to. When you come over to us, maybe we can go over this more in depth. We are keeping this branch and the pull request open in the meantime.

**Pull request merged**